packages/flutter_tools/test/android_device_test.dart

```diff
@@ -41,11 +42,25 @@ emulator-5612          host features:shell_2
 
     testUsingContext('android n', () {
       List<AndroidDevice> devices = getAdbDevices(mockAdbOutput: '''
+List of devices attached
 ZX1G22JJWR             device usb:3-3 product:shamu model:Nexus_6 device:shamu features:cmd,shell_v2
 ''');
       expect(devices, hasLength(1));
       expect(devices.first.name, 'Nexus 6');
     });
+
+    BufferLogger logger = new BufferLogger();
```

Comment:
Good point. I don't need to create a new logger... just need to use `testLogger` to access the logger already created by `testUsingContext`. Fixed.